src/app/static/src/tests/components/projects/projectHistory.test.ts

```diff
@@ -534,6 +560,33 @@ describe("Project history component", () => {
         }
     });
 
+    it("can use carriage return to invoke renameProject action", async () => {
+        if (switches.renameProject) {
```

Comment:
Shouldn't need this

Reply:
I am not totally sure what you meant here. Are you saying the enter key should not be tested ?

Reply:
No I'm saying just take out that conditional on the switches